plugins/plugin-site/src/templates/plugin_healthScore.jsx

```diff
@@ -0,0 +1,41 @@
+import {graphql} from 'gatsby';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {PropTypesJenkinsPlugin} from '../proptypes';
+
+import PluginPageLayout from '../components/PluginPageLayout';
+import PluginHealthScore from '../components/PluginHealthScore';
+
+const TemplatePluginHealthScore = ({data: {jenkinsPlugin: plugin, healthScore}}) => {
+    return (
+        <PluginPageLayout plugin={plugin}>
+            <PluginHealthScore healthScore={healthScore}/>
+        </PluginPageLayout>
+    );
+};
+
+TemplatePluginHealthScore.displayName = 'PluginPage';
+TemplatePluginHealthScore.propTypes = {
+    data: PropTypes.shape({
+        jenkinsPlugin: PropTypesJenkinsPlugin.isRequired,
+        healthScore: PropTypes.shape({
+        }).isRequired
+    })
+};
+
+/* eslint no-undef: "off" */
+export const pageQuery = graphql`
+    query ($name: String!) {
+        jenkinsPlugin(name: {eq: $name}) {
+            ...JenkinsPluginFragment
+        }
+
+        healthScore: allJenkinsPluginHealthScore(filter: {name: {eq: $name}}) {
```

Comment:
all prefix returns an array. I think you just want `jenkinsPluginHealthScore(name: {eq: $name})`

Reply:
also if you goto localhost:3000/__graphql you get the full graphql explorer.

Reply:
actually, jenkinsPlugin actually links them
https://github.com/jenkins-infra/plugin-site/blob/master/plugins/gatsby-source-jenkinsplugins/gatsby-node.js#L41
```
        jenkinsPlugin(name: {eq: $name}) {
            ...JenkinsPluginFragment
            healthScore: {
            }
        }
```

Reply:
thank you! This is what I didn't understand!

Reply:
I think I didn't structure the data in the API correctly because I would need to describe all the graphql for any new  `details` when this should be easily extensible. I need to change the `object` structure for an `array` so that we can easily create a table.

Reply:
the source plugin can also do some data mashing for you, and graphql can do some joins. So like if you want to do one dataset for detail definitions, and one dataset for the actual results, we can do a join.

i'm just not sure if you have a question or not. Feel free to ping me again.